### PR TITLE
wxGUI: fix i18n issues in r.li.setup gui

### PR DESCRIPTION
**Pull request by slichtzzz**
Fix for i18n issues

### CODE REVIEW

File: gui/wxpython/rlisetup/wizard.py [hunk 1]
```diff
@@ -1730,16 +1723,8 @@ def SampleFrameChanged(self, region):
...
-                    "Draw Sampling "
-                    + drawtype
-                    + " "
-                    + str(self.regioncount)
-                    + " of "
-                    + str(self.numregions)
-                )
-            )
+            self.title.SetLabel(_("Draw Sampling {draw_type} {count} of {num_regions}").format(draw_type=drawtype, count=self.regioncount, num_regions=self.numregions))
+
```
Comment (petrasovaa) [inline gui/wxpython/rlisetup/wizard.py:1727]:
```suggestion
            self.title.SetLabel(_("Draw sampling {draw_type} {count} of {num_regions}").format(draw_type=drawtype, count=self.regioncount, num_regions=self.numregions))
```

File: gui/wxpython/rlisetup/wizard.py [hunk 2]
```diff
@@ -1832,14 +1817,8 @@ def afterRegionDrawn(self):
...
-            self.title.SetLabel(
-                _(
-                    "Select sample area "
-                    + str(self.areascount + 1)
-                    + " of "
-                    + str(self.areanum)
-                )
-            )
+            areascount1 = self.areascount + 1
+            self.title.SetLabel(_("Select sample area {areas_count} of {area_num}").format(areas_count=areascount1, area_num=self.areanum))
```
Comment (petrasovaa) [inline gui/wxpython/rlisetup/wizard.py:1821]:
```suggestion
            self.title.SetLabel(_("Select sample area {areas_count} of {area_num}").format(areas_count=self.areascount + 1, area_num=self.areanum))
```

**Comment by petrasovaa** [comment]
Thanks for the PR, I added some suggestions. Also could you please apply black to reformat the long lines?

**Comment by slichtzzz** [comment]
I tried to fix everything as you suggested. Thank you very much for helping me get done real programming stuff! I am a professional translator, but programmer at heart and by education.

Hope everything is fine now :-)

File: gui/wxpython/rlisetup/wizard.py [hunk 3]
```diff
@@ -1833,11 +1828,8 @@ def afterRegionDrawn(self):
             return True
         else:
             self.title.SetLabel(
-                _(
-                    "Select sample area "
-                    + str(self.areascount + 1)
-                    + " of "
-                    + str(self.areanum)
+                _("Select sample area {areas_count} of {area_num}").format(
+                    areas_count=areascount + 1, area_num=self.areanum
```
Comment (petrasovaa) [inline gui/wxpython/rlisetup/wizard.py:1832]:
```suggestion
                    areas_count=self.areascount + 1, area_num=self.areanum
```

Comment (slichtzzz) [reply gui/wxpython/rlisetup/wizard.py]: I uploaded the changes

File: gui/wxpython/rlisetup/wizard.py [hunk 4]
```diff
@@ -1731,15 +1728,13 @@ def SampleFrameChanged(self, region):
...
-                    + str(self.regioncount)
-                    + " of "
-                    + str(self.numregions)
+                _("Draw sampling {draw_type} {count} of {num_regions}").format(
+                    draw_type=drawtype,
+                    count=self.regioncount,
+                    num_regions=self.numregions,
                 )
             )
+
```
Comment (petrasovaa) [inline gui/wxpython/rlisetup/wizard.py:1737]:
```suggestion
```

Comment (slichtzzz) [reply gui/wxpython/rlisetup/wizard.py]: Hello! I removed the line and committed changes. Hope, now it's fine)

**Review by petrasovaa — APPROVED** [review]

**Comment by petrasovaa** [comment]
Looks good now, thank you!

**Comment by slichtzzz** [comment]
Thanks and congratulations with the forthcoming International Women's Day!

**Pull request merged**